docker-compose.yml

```diff
@@ -123,7 +119,7 @@ services:
       dockerfile: Dockerfile
       args:
         PACKAGE: headless
-    image: firezone-headless
+    image: firezone/client
```

Comment:
`elixir/README.md` should be changed to use new container names.

Reply:
Hmm, these image names don't get used as the service names though for `docker-compose`. These are only so that the `docker compose up -d` will find the images we built in the `build-images` workflow.

Reply:
I project-searched those strings and didn't see anything pop up...

Reply:
I double-checked the readme and looks like indeed we don't need to change anything, so no worries.